packages/web/components/Form/FormWizard.js

```diff
@@ -236,7 +263,11 @@ const FormWizard = ({ steps, currentStep, onSubmit, onPrev, data, defaultValues,
 							<StepItem completed={i <= currentStepIndex} key={step.slug}>
 								<div>
 									<StepNumber>{showIcon ? <Icon /> : i + 1}</StepNumber>
-									<StepLabel>{step.label}</StepLabel>
+									<Link
+										href={`/technology/${data?.technology?.id}/edit/${step.slug}`}
```

Comment:
Just to make sure that this is the expected behavior: if I'm still creating the technology (it doesn't exist in the database yet), on the first step and try to click on some of the following steps links, I'm redirected to the first page. The reason why is we have verification in the `getInitialProps` in the `edit.js` page. 

Is it the expected behavior?

Reply:
@lcnogueira To my understanding; yes, that's the expected behavior.

Reply:
Got it